### PR TITLE
Stunprods no longer go into backpacks

### PR DESCRIPTION
**Pull request by lzimann**
:cl: Lzimann
tweak: Stunprods no longer fit in your backpack, they go on your back now.
/:cl:

Also changes the charge consumed per hit to 2000 from 2500(gives one extra hit) to slightly compensate.

Sprites by Lexorion: 

qrevyby

### CODE REVIEW

File: code/game/objects/items/weapons/teleprod.dm [hunk 1]
```diff
@@ -1,9 +1,11 @@
 /obj/item/weapon/melee/baton/cattleprod/teleprod
 	name = "teleprod"
 	desc = "A prod with a bluespace crystal on the end. The crystal doesn't look too fun to touch."
+	w_class = 3
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/teleprod.dm]: :cop:
:alarm_clock: 
# UNDOCUMENTED CHANGES
:alarm_clock: 
:cop:

**Comment by Ultimate-Chimera** [comment]
rip

File: code/game/objects/items/weapons/teleprod.dm (hunk 1 above)
Comment (lzimann) [inline code/game/objects/items/weapons/teleprod.dm]: Undocumented changes would be me not doing this. This leaves teleprods as they were before.

**Comment by InsaneHyena** [comment]
This kills tactical stun prods. I always carry one in my bag in order to catch antags by surprise and get valids.

**Comment by WJohn** [comment]
Well, it's more consistent. Didn't make sense how stunprods could fit into your backpack but spears couldn't.

**Comment by MMMiracles** [comment]
As already said, people will just carry around unfinished prods for about the same effect. Its better than outright having an active prod ready to go I guess so its a step in the right direction.

**Comment by AdamElTablawy** [comment]
make unfinished prods/wired rods also not fit

**Comment by Ohlos** [comment]
Nerfbus on the run.

**Comment by Fox-McCloud** [comment]
Don't think this is a good idea, personally.

It's going to mean people are more apt to engage in lethal force to diffuse a situation rather than just stunning them and running or the likes.

This may cut down on validhunting a little bit, but, ultimately, I think this is going to create more problems than it solves. 

It also further pushes more power to science, who will still be able to run around with it concealed without a care in the world, thanks to Bags of Holding.

Andd as others keep mentioning, people will just put the parts in their backpack and build this when it's really needed (people already due this to avoid getting arrested).

There's got to be a more creative way of nerfing these.

**Comment by MMMiracles** [comment]
@Fox-McCloud I think if science has multiple people running around with bags of holding, the least of your concern from that department is a scientist with a stun prod.

**Comment by Cheridan** [comment]
>Fox thinks that people use stunprods non-lethally 

lol

**Comment by X-TheDark** [comment]
The standard modus operandi for stunprods is having them active in the backpack (and have the backpack open), then just click-attack to stun and bag the poor sucker. So this is good, in my opinion.

**Comment by KorPhaeron** [comment]
In the last 30 days the following objects were made:

/obj/item/weapon/melee/baton/cattleprod 335

In the last 30 days (720 hours), the server has been up for:

Rounds played: 828

I'm not sure how this PLAGUE OF STUNPRODS thing started up again bad enough to lead to multiple prs about it.

I'm not saying the item is super amazing design wise but a single person building one every 2 rounds is hardly the issue the forums are making it out to be.

**Comment by InsaneHyena** [comment]
> one stun prod was made every two rounds

That does not sound right

File: code/game/objects/items/weapons/teleprod.dm (hunk 1 above)
Comment (Supermichael777) [inline code/game/objects/items/weapons/teleprod.dm]: for what reason
they should logically be just as large

**Comment by Supermichael777** [comment]
no one has ever stun and run
they kill

**Comment by AdamElTablawy** [comment]
it's either one stunprod every two rounds, or more likely, multiple stunprods crafted in a single round and other rounds where stunprods would be generally useless not crafted.

See-stunprods being built more often on rev/traitor vs stunprods being built on cult/APE ESCAPE.

Not to mention that even if it's  not 999999 stunprods every five seconds flying out of an assistant's asshole, it's still...bad.

**Comment by evsey9** [comment]
If a stunprod  doesn't fit into a backpack, then why do 50 rods do?
Or even one.

**Comment by Iamgoofball** [comment]
@evsey9 balance

**Comment by Iamgoofball** [comment]
According to our Stat Gathering(tm)
http://ss13.eu/tgdb/tg/latest_stats.html#objectsmade

/obj/item/weapon/melee/baton/cattleprod (Stunprods) have been made 335 times.

File: code/game/objects/items/weapons/teleprod.dm (hunk 1 above)
Comment (lzimann) [inline code/game/objects/items/weapons/teleprod.dm]: because i don't have sprites for it and i don't think i've ever seen a teleprod being used

**Comment by AnturK** [comment]
That only counts the crafting prods, not manually assembled ones afaik so this might be not really representative.

**Comment by PKPenguin321** [comment]
There are definitely more prods made via slapcrafting than personal crafting too

**Comment by Iamgoofball** [comment]
See this is why we stop supporting slapcraft for fucking our stay gathering

**Comment by lzimann** [comment]
but then we would have to use the crafting menu which is bad

**Comment by Iamgoofball** [comment]
@MrStonedOne did you ever make any progress on that recode of slapcrafting

**Comment by InsaneHyena** [comment]
Oh, it doesn't count slapcrafting? That explains it, I always slapcraft if I can.

**Comment by Fox-McCloud** [comment]
And then everyone resorted to welding someone's face off if they so much as disarmed them more than three times.

**Comment by lzimann** [comment]
at least it's not a stun which you can't do anything about

**Comment by TheNightingale** [comment]
Implying they didn't do that after stunning them anyway.

**Comment by TechnoAlchemisto** [comment]
Expect more ahelps for critting people to get them out of departments when they're being rowdy tbh

**Comment by Fox-McCloud** [comment]
My bigger concern is that this is going to make murderbone as a strategy even more viable--it won't be *quite* as feasible for people to carry one of these around for self-defense "Just in case", meaning...there's even less opposition for the "Kill everyone" types outside of security.

Of course, if the intentions is of going full Goon (no stun weapons or ranged delivery systems for non-antags outside of security), then, this is a step in the "right" direction.

**Pull request merged**